### ISSUE TITLE
GPD Pocket 3 rotate issues

### ISSUE THREAD

**Issue by mifritscher** [wimpysworld/umpc-ubuntu#31]
At least on my pocket 3, the program wants to rotate pen and stylus right. But I need the first transformation matrix to get the touch screen right in normal laptop mode:

`xinput set-prop "GXTP7380:00 27C6:0113" "Coordinate Transformation Matrix" 1 0 0 0 1 0 0 0 1`

Additionally, I get no further message even when rotating the device. A hacked
`  fprintf(stderr, "%d %d %d\n",x,y,z);` gets always
`50 0 -54`

the raw data:
202 -3 -216
.

**Comment by gdevenyi** [comment]
I had the same experience. The initial setup (laptop mode) doesn't properly transform the touch/pen input.

**Comment by JohnJayMcKaye** [comment]
I have the same problem, pen input doesn't rotate and the pen button must be hold to scribe/ drag some window

**Comment by eastpiger** [comment]
Same problem here. Actually, after upgrading the kernal, rotation and touch screen works good as expected. The problem happens after run this script. Seems we rotate the framebuffer but failed to rotate the touchscreen together correctly.

I fixed it by comment out these lines:
```

  # Rotate the touchscreen.
  inject_data "${TOUCH_RULES}"
  # Reload udev rules
  udevadm control --reload-rules
  udevadm trigger
```

**Comment by flexiondotorg** [comment]
Fixed via #40
